docs/src/main/sphinx/release/release-378.md

```diff
@@ -0,0 +1,96 @@
+# Release 378 (20 Apr 2022)
+
+## General
+
+* Improve planning performance of queries with large `IN` lists.
+  ({issue}`11902`, {issue}`11918`, {issue}`11956`)
+* Fix reporting of total spilled bytes in JMX metrics. ({issue}`11983`)
+
+## Security
+
+* Require value for the ``internal-communication.shared-secret`` configuration
```

Comment:
@dain I moved this to Security instead of General and reworded .. ok?

Reply:
Also .. @martint and @dain .. for many users this is a breaking change. How do we best message this? I am going to mention it in the broadcast on Thursday but beyond that?